templates/idx.html

```diff
@@ -63,236 +114,586 @@
                 </div>
             </article>
             <div class="content">
-                <p>
-                    <strong>Try Enarx</strong> allows you to run a <strong>WebAssembly</strong> workload in an encrypted
-                    <strong>Enarx Keep</strong>.
-                    You can either <strong>select a demo</strong> to see it working (from <strong>Drawbridge</strong>,
-                    an Enarx package registry) or <strong>upload a Wasm file</strong> along with <a
-                        href="https://enarx.dev/docs/running/enarx_toml" target="_blank">the Enarx runtime
-                        configuration</a>.
-                </p>
-                <p>
-                    {% if star %}
-                    Thanks for starring the Enarx project!
-                    {% else %}
-                    To raise the workload limits, star <a href="https://github.com/enarx/enarx">the Enarx project</a>.
-                    {% endif %}
-                </p>
+                <div class="has-text-centered is-size-3">
+                    <strong>Try Enarx</strong> allows you to run a <a href="https://webassembly.org">WebAssembly</a>
```

Comment:
I'll do this for all links. It's kinda abusive to force new tabs but it will help keep the page open and prevent them from accidentally leaving on mobile or something.